report_xml/models/ir_actions_report.py

```diff
@@ -29,6 +29,12 @@ class IrActionsReport(models.Model):
         help='Add `<?xml encoding="..." version="..."?>` at the start of final report '
         "file.",
     )
+    xml_extension = fields.Selection(
```

Comment:
Do it a Char field for not having to program an extension module for further ones.